src/model/genesis/AGenesis.java

```diff
@@ -277,4 +278,38 @@ public int maxGeneration(){
 		return this.maxGen;
 	}
 
+	private void meetPeople(Random r) {
+		// efficiency... I got nothing. so this is a stopgap until we can talk about it some.
+		List<IPerson> people = livingPopulation();
+
+		for (int attempt = 0; attempt < MEETING_ATTEMPTS_PER_YEAR; attempt++) {
+			int max = people.size()/2;
+			Collections.shuffle(people);
+			for (int i = 0; i < max; i++) {
+				IPerson p1 = people.get(i);
+				IPerson p2 = people.get(i * 2);
```

Comment:
In the i=0 case, wouldn't this result in p1 and p2 being the same person? Why did you choose to do i * 2?

Reply:
because I'm an idiot sometimes. I will change that.